backend/src/validation-schemas/episode/create-episode/create-episode.validation-schema.ts

```diff
@@ -10,6 +11,13 @@ const episodeCreate = common.keys({
     'number.required': EpisodeValidationMessage.PODCAST_ID_REQUIRE,
     'number.integer': EpisodeValidationMessage.PODCAST_ID_NUMBER_FORMAT,
   }),
+  [EpisodePayloadKey.STATUS]: Joi.when(EpisodePayloadKey.RECORD_DATA_URL, {
```

Comment:
we should have the same logic on the frontend & backend
let's move it to shared 🤔

Reply:
We can't have same logic, becouse on the front we have to validate in a bundle status, filelist and dataUrl , but on the front we validate only dataUrl and status.

Reply:
got you